### PR TITLE
tweak readme

### PR DESCRIPTION
**Pull request by dbu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,15 +22,17 @@ Require the library using composer:
 
 ## Bootstrapping
 
+You will need to register for a Rokka.io account and use the api key you receive. The recommended way to do this is by downloading the [rokka-cli](https://github.com/rokka-io/rokka-client-php-cli/releases).
```
Comment (thePanz) [inline README.md:25]:
```suggestion
You will need to register for a Rokka.io account and use the api key you receive.
The recommended way to do this is by using the [rokka-cli](https://github.com/rokka-io/rokka-client-php-cli/releases).
```

File: README.md [hunk 2]
```diff
@@ -54,30 +56,33 @@ $apiKey = 'myKey';
 $imageClient = Factory::getImageClient($organization, $apiKey);
 ```
 
-There is an optional fourth parameter to specify the base URL of the Rokka API. This usually does not need to be adjusted.
+There is an optional parameter to specify the base URL of the Rokka API. This usually does not need to be adjusted.
```
Comment (thePanz) [inline README.md]:
```suggestion
There is an optional parameter to specify the base URL of the Rokka API.
This usually does not need to be adjusted.
```

File: README.md (hunk 1 above)
Comment (thePanz) [inline README.md]:
```suggestion
You will need to register for a Rokka.io account and use the API key you receive.
The recommended way to do so is by using the [rokka-cli](https://github.com/rokka-io/rokka-client-php-cli/releases).
```

File: README.md [hunk 3]
```diff
@@ -54,30 +56,33 @@ $apiKey = 'myKey';
...
 ]
-
 ```
 
 ## Usage
 
-See the [official documentation](https://rokka.io/documentation) on how to use the Rokka API.
+Read the [Getting Started](https://rokka.io/documentation/guides/get-started.html) guide of the [rokka.io documentation](https://rokka.io/documentation) to learn about the basic concepts of rokka.
+
+The image client and user client provide the operations described in the API Reference section of the rokka documentation.
```
Comment (thePanz) [inline README.md]:
```suggestion
The image and user clients provide the operations described in the API Reference section of the rokka documentation.
```

File: README.md [hunk 4]
```diff
@@ -54,30 +56,33 @@ $apiKey = 'myKey';
...
 
-## API Docs
+For example, to upload an image, you would do:
 
-See als the [API Docs](https://rokka.io/client-php-api/master/) for further information.
+```php
+
```
Comment (thePanz) [inline README.md:82]: Is the example missing here? :)

**Review by thePanz — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (dbu) [reply README.md:82]: hm. this was some old changes i found on my disk... i will try to figure out what i intended to do here

Comment (dbu) [reply README.md:82]: guess i wanted to add an example. i removed this bit and add back the link to the api docs to be able to merge this as the rest of the changes would be useful.

**Pull request merged**